setup.py

```diff
@@ -364,12 +377,18 @@ def parse_jsargs():
     print("Installing Bokeh:")
     if pth_removed:
         print("  - removed path file at %s" % path_file)
-    print("  - using %s built bokehjs from bokehjs/build\n" % ("NEWLY" if jsbuild else "PREVIOUSLY"))
+    if jsinstall:
+        print("  - using %s built BokehJS from bokehjs/build\n" % ("NEWLY" if jsbuild else "PREVIOUSLY"))
+    else:
+        print("  - using 'PACKAGED' built BokehJS\n")
 
 elif '--help' in sys.argv:
-    print("Bokeh-specific options available with 'install' or 'develop':\n")
-    print("  --build_js          build and install a fresh BokehJS")
-    print("  --install_js        install only last previously built BokehJS")
+    if jsinstall:
+        print("Bokeh-specific options available with 'install' or 'develop':\n")
+        print("  --build_js          build and install a fresh BokehJS")
+        print("  --install_js        install only last previously built BokehJS")
+    else:
+        print("Bokeh using 'PACKAGED' built BokehJS\n")
```

Comment:
You mean `Bokeh using built BokehJS from 'bokeh.server.static'` without the PACKAGED? or other wording? I accept suggestion :wink: